### DISCUSSION TITLE
Is it possible to influence the spatial distribution of phases

### DISCUSSION THREAD

**Discussion by hem-28** [kip-hart/MicroStructPy#59] [Q&A] [answered]
Hello,

Is it possible to influence spatial distribution of phases within the domain? If so could you please point me to an example?

Thanks

**Comment by kip-hart** [comment, marked as answer]
Hi yes its possible, here is the documentation for specifying spatial distributions for each phase:

https://docs.microstructpy.org/en/latest/cli/material.html#grain-position-distribution

Let me know I can help further.
